week2-6.6-decorator_factory.py

```diff
@@ -0,0 +1,44 @@
+from functools import wraps
+
+
+class MsgError(Exception):
```

Comment:
It's bad practice to leave a class unimplemented.
Read about how to implement a custom error class in a good way.